lasagne/layers/conv.py

```diff
@@ -117,23 +109,22 @@ class Conv1DLayer(Layer):
         An integer or a 1-element tuple specifying the stride of the
         convolution operation.
 
-    border_mode : str, one of 'valid', 'full', 'same'
-        A string indicating the convolution border mode.
+    pad : {0, (0,), 'full', 'same'} (default: 0)
```

Comment:
> wrapping the input in a `PadLayer`

Interesting idea. Might be a little unexpected to get two layers when you thought you were instantiating only one, though. We could instead just call `theano_extensions.padding.pad` as part of `get_output_for()`, that would fix the documentation problem and just be terribly slow until Theano catches up.